utils/doclint/generateDotnetApi.js

```diff
@@ -73,7 +73,7 @@ documentation.setLinkRenderer(item => {
 // get the template for a class
 const template = fs.readFileSync(path.join(__dirname, 'templates', 'interface.cs'), 'utf-8');
 
-// map the name to a C# friendly one (we prepend an I to denote an interface)
+// map the name to a .NET friendly one (we prepend an I to denote an interface)
```

Comment:
```suggestion
// map the name to a C# friendly one (we prepend an I to denote an interface)
```

Minor nit: we still generate C#, so I'd keep that.